### PR TITLE
faq: add info for schema is changed error

### PR DESCRIPTION
**Pull request by zimulala**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: faq/sql-faq.md [hunk 1]
```diff
@@ -178,11 +178,12 @@ TiDB 支持改变 [per-session](/system-variables.md#tidb_force_priority)、[全
 
 TiDB 在执行 SQL 语句时，会使用当时的 `schema` 来处理该 SQL 语句，而且 TiDB 支持在线异步变更 DDL。那么，在执行 DML 的时候可能有 DDL 语句也在执行，而你需要确保每个 SQL 语句在同一个 `schema` 上执行。所以当执行 DML 时，遇到正在执行中的 DDL 操作就可能会报 `Information schema is changed` 的错误。为了避免太多的 DML 语句报错，已做了一些优化。
 
-现在会报此错的可能原因如下（后两个报错原因与表无关）：
+现在会报此错的可能原因如下（只有第一个报错原因与表有关）：
 
 - 执行的 DML 语句中涉及的表和集群中正在执行的 DDL 的表有相同的，那么这个 DML 语句就会报此错。
 - 这个 DML 执行时间很久，而这段时间内执行了很多 DDL 语句，导致中间 `schema` 版本变更次数超过 1024 （此为默认值，可以通过 `tidb_max_delta_schema_count` 变量修改）。
 - 接受 DML 请求的 TiDB 长时间不能加载到 `schema information`（TiDB 与 PD 或 TiKV 之间的网络连接故障等会导致此问题），而这段时间内执行了很多 DDL 语句，导致中间 `schema` 版本变更次数超过 100。
+- TiDB 重启后遇到第一个 DDL 操作期间，执行 DML 并且在它执行过程中遇到了 DDL 操作，那么这个 DML 很可能会报此错。
```
Comment (TomShawn) [inline faq/sql-faq.md:186]: “第一个 DDL 操作”和“执行过程中遇到了 DDL 操作”，这两个是同一个 DDL 操作吗？

Comment (zimulala) [reply faq/sql-faq.md:186]: 可能遇到多个 DDL，第一个 DDL 一定会包含。我改一下描述

Comment (TomShawn) [reply faq/sql-faq.md]: 能否请一位研发老师看看呢？谢谢~

Comment (zimulala) [reply faq/sql-faq.md]: PTAL @tiancaiamao @qw4990

**Review by qw4990 — APPROVED** [review]

File: faq/sql-faq.md [hunk 2]
```diff
@@ -178,11 +178,12 @@ TiDB 支持改变 [per-session](/system-variables.md#tidb_force_priority)、[全
 
 TiDB 在执行 SQL 语句时，会使用当时的 `schema` 来处理该 SQL 语句，而且 TiDB 支持在线异步变更 DDL。那么，在执行 DML 的时候可能有 DDL 语句也在执行，而你需要确保每个 SQL 语句在同一个 `schema` 上执行。所以当执行 DML 时，遇到正在执行中的 DDL 操作就可能会报 `Information schema is changed` 的错误。为了避免太多的 DML 语句报错，已做了一些优化。
 
-现在会报此错的可能原因如下（后两个报错原因与表无关）：
+现在会报此错的可能原因如下（只有第一个报错原因与表有关）：
 
 - 执行的 DML 语句中涉及的表和集群中正在执行的 DDL 的表有相同的，那么这个 DML 语句就会报此错。
 - 这个 DML 执行时间很久，而这段时间内执行了很多 DDL 语句，导致中间 `schema` 版本变更次数超过 1024 （此为默认值，可以通过 `tidb_max_delta_schema_count` 变量修改）。
 - 接受 DML 请求的 TiDB 长时间不能加载到 `schema information`（TiDB 与 PD 或 TiKV 之间的网络连接故障等会导致此问题），而这段时间内执行了很多 DDL 语句，导致中间 `schema` 版本变更次数超过 100。
+- TiDB 重启后遇到第一个 DDL 操作期间，执行 DML 并且在它执行过程中遇到了第 1 个 DDL 操作（即在第 1 个 DDL 执行前此 DML 对应事务 `begin`，在此 DDL 第一个 `schema` 版本变更后此 DML 对应事务 `commit`），那么这个 DML 会报此错。
```
Comment (TomShawn) [inline faq/sql-faq.md:186]:
```suggestion
- TiDB 重启后执行第一个 DDL 操作前，执行 DML 操作，并且在执行过程中遇到了第 1 个 DDL 操作（即在执行第 1 个 DDL 操作前，启动该 DML 对应的事务，且在该 DDL 变更第一个 `schema` 版本后，提交该 DML 对应的事务），那么这个 DML 会报此错。
```

**Comment by zimulala** [comment]
PTAL @TomShawn

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**